_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,656 @@
+  how the current transaction relay works (called fanout). In these posts, he
+  discusses different results that he found while developing Erlay, such as
+  [filtering based on transaction knowledge][erlay knowledge] not mattering as
+  much as expected. He also experimented with selecting [how many peers should
+  receive a fanout][erlay fanout amount], and found that there was a 35%
+  bandwidth savings with 8 outbound peers and 45% with 12 outbound peers, but
+  also found a 240% increase in latency. In his two other experiments, he
+  determined the [fanout rate based on how a transaction was
+  received][erlay transaction received] and [when to select canidate
+  peers][erlay candidate peers]. These experiments, which combined fanout and
+  reconciliation, helped him determine when to use each method.
+
+{:#lneas}
+- **LN ephemeral anchor scripts:** ...
+
+{:#probpayments}
+- **Probabilistic payments:** ...
+
+<div markdown="1" class="callout" id="vulns">
+
+## Summary 2025: Vulnerabilities
+
+...
+
+</div>
+
+## March
+
+{:#forkingguide}
+- **Bitcoin Forking Guide:** Anthony Towns [posted][fork guide post] to Delving
+  Bitcoin a guide on how to build community consensus for changes to Bitcoin’s
+  consensus rules. According to Towns, the process of establishing consensus can
+  be divided in four steps, namely [research and development][fork guide red],
+  [power user exploration][fork guide pue], [industry evaluation][fork guide
+  ie], and [investor review][fork guide ir]. However, Towns warned readers that
+  the guide aims to be only a high-level procedure, and that it could work only
+  in a cooperative environment.
+
+{:#templatemarketplace}
+- **Private block template marketplace to prevent centralizing MEV:** Developers
+  Matt Corallo and 7d5x9 [posted][template mrkt post] to Delving Bitcoin a
+  proposal that would allow to prevent a future in which MEVil — a form of MEV
+  extraction leading to mining centralization — proliferates on Bitcoin. The
+  proposal, referred to as [MEVpool][mevpool gh], would allow parties to bid in
+  public markets for selected space within miner block templates (i.e. "I’ll pay
+  X [BTC] to include transaction Y as long as it comes before any other
+  transaction which interacts with the smart contract identified by Z").
+
+  While today services of preferential transaction ordering within block
+  templates are expected to be provided only by large miners, leading to
+  centralization, a trust-reduced public market would allow any miner to work on
+  blinded block templates whose complete transactions aren’t revealed to miners
+  until they’ve produced sufficient proof of work to publish the block.
+
+  The authors warned that this proposal would require multiple marketplaces to
+  compete to help preserve decentralization against the dominance of a single
+  trusted marketplace.
+
+{:#lnupfrontfees}
+- **LN upfront and hold fees using burnable outputs:** John Law [posted][ln fees post]
```

Comment:
Similar recommendation to above to link to our own previous coverage. Also suggest more original digest summaries vs repeating what was originally covered @TumaBitcoiner